Source/WebKitLegacy/Storage/WebStorageNamespaceProvider.h

```diff
@@ -49,11 +50,14 @@ class WebStorageNamespaceProvider final : public WebCore::StorageNamespaceProvid
 private:
     explicit WebStorageNamespaceProvider(const String& localStorageDatabasePath);
 
-    Ref<WebCore::StorageNamespace> createSessionStorageNamespace(WebCore::Page&, unsigned quota) override;
     Ref<WebCore::StorageNamespace> createLocalStorageNamespace(unsigned quota, PAL::SessionID) override;
     Ref<WebCore::StorageNamespace> createTransientLocalStorageNamespace(WebCore::SecurityOrigin&, unsigned quota, PAL::SessionID) override;
 
+    RefPtr<WebCore::StorageNamespace> sessionStorageNamespace(WebCore::SecurityOrigin&, WebCore::Page&, ShouldCreateIfNotExists) override;
```

Comment:
Could be marked as final

Reply:
Done - although I'm not sure why this can be final and the others above it aren't, but I'm not opposed to making this final.